docs/api.rst

```diff
@@ -0,0 +1,13 @@
+.. _api_documentation:
+
+=================
+API Documentation
+=================
+
+.. currentmodule:: dicodile
```

Comment:
For the API, I think that for now we should only expose the description for the `dicodile` function inside this module, as it is the main one that will be used.